roll.py

```diff
@@ -23,12 +24,16 @@ def role_legueoflegends(lane) -> str:
     if(lane.lower() not in LEAGUE_POSITIONS):
         lane = LEAGUE_POSITIONS[random.randint(0, len(LEAGUE_POSITIONS) - 1)]
 
+def build_leagueoflegends(build) -> str:
+    if (build.lower() not in LEAGUE_BUILD):
+        build = LEAGUE_BUILD[random.randint(0, len(LEAGUE_BUILD) - 1)]
+
     champList = LEAGUE_CHAMPIONS[lane.lower()]
 
     randChampIndex = random.randint(0, len(champList) - 1)
     pickedChamp = champList[randChampIndex]
 
-    return lane + ' ' + pickedChamp
+    return lane + ' ' + pickedChamp + ' ' + build
```

Comment:
`lane` does not exist in this function. Consider this:
```suggestion
    return  pickedChamp + ' ' + build
```